src/Listener/UpdatedAt.php

```diff
@@ -22,4 +24,12 @@ public function __invoke(OnUpdate $event): void
             $event->command->registerAppendix($this->field, $event->timestamp);
         }
     }
+
+    #[Listen(OnCreate::class)]
+    public function onCreate(OnCreate $event): void
+    {
+        if ($this->nullable === false && !isset($event->state->getData()[$this->field])) {
+            $event->state->register($this->field, $event->timestamp);
+        }
```

Comment:
Should work like the createdAt behavior
```suggestion
        if (!$this->nullable) {
            $event->state->register($this->field, $event->timestamp);
        }
```